### DISCUSSION TITLE
3rd Ambient temp option in Roast Properties

### DISCUSSION THREAD

**Discussion by BCRoaster** [artisan-roaster-scope/artisan#1717] [General]
When I save a profile, I want the statistics box to reflect the ambient temp in the room where I roast. I noticed in Roast/Roast Properties only two options, BT or ET. I don't understand as neither of those are ambient temps. Currently, I edit post-roast, enter the actual ambient temp, and save the file. Could there not be a third option in the drop box?

**Comment by roasterdave** [comment]
Extra devices, when defined, also show in the drop down.  If you have another temperature sensor you can define an extra device to record it.  At roast time select that device in the Ambient Source drop down and click Update.  However, this is still a manual process.  

An automatic solution can be configured on the Config>> Device>> Ambient tab.  Artisan will automatically populate the ambient conditions fields for each roast from the device(s) when you click ON (or START if you skipped ON).  To use this feature requires using the predefined device(s).  A Phidget TMP1000 reports ambient temperature only, the HUM100x reports ambient temperature and humidity, the PRE1000 reports only pressure, or the Yoctopuce Meteo reports all three.  Be aware that these ambient readings are done once per roast.  To continuously monitor and record ambient conditions requires setting up and using an extra device.
